StreetTreesFoundationKit/STFKLocationRequest.swift

```diff
@@ -56,13 +56,13 @@ public class STFKLocationRequest: NSObject, CLLocationManagerDelegate {
     //******************************************************************************************************************
     // MARK: - CLLocationManager Delegate
     
-    public func locationManager(manager: CLLocationManager, didFailWithError error: NSError) {
+    open func locationManager(_ manager: CLLocationManager, didFailWithError error: Error) {
         self.stopUpdating()
-        self.handler(location: nil, error: error)
+        self.handler(nil, error as NSError?)
```

Comment:
The handler method calls specifically for an optional CLLocation and optional NSError objects.  Passing in a Swift Error object only causes a syntax error.  Casting it as an NSError clears the syntax error.